cves/CVE-2013-2115.yml

```diff
@@ -162,37 +184,43 @@ lessons:
     If you think of another lesson we covered in class that applies here, feel
     free to give it a small name and add one in the same format as these.
   defense_in_depth:
-    applies:
+    applies: false
     note:
   least_privilege:
-    applies:
-    note:
+    applies: true
+    note: |
+      if the OGNL successfully got through, the privileges of the user
```

Comment:
The 'i' in 'if' should probably be capitalized to be consistent with the rest of the document.